server/models/Member.js

```diff
@@ -110,6 +111,18 @@ export default function(Sequelize, DataTypes) {
           };
         },
       },
+      hooks: {
+        afterCreate: instance => {
+          if (instance.role !== roles.FOLLOWER) {
+            invalidateContributorsCache(instance.CollectiveId);
+          }
+          return null;
+        },
+        afterUpdate: instance => {
+          invalidateContributorsCache(instance.CollectiveId);
```

Comment:
Can we add the same `if (instance.role !== roles.FOLLOWER)` check here?

Reply:
I thought about a case, not sure it is valid. I was wondering in the case that role of the member is updated to `FOLLOWER`, wouldn't we want to update the contributors to account for the change?

Reply:
We don't use the role `FOLLOWER` yet (it will be implemented with https://github.com/opencollective/opencollective/issues/1445) but I don't think that we'll have such cases, if an existing member wants to follow a collective then we'll probably create a new `Member` entry rather than updating the existing one.